cmd/tendermint/commands/reset_priv_validator.go

```diff
@@ -28,25 +30,38 @@ func init() {
 // XXX: this is totally unsafe.
 // it's only suitable for testnets.
 func resetAll(cmd *cobra.Command, args []string) {
-	resetPrivValidator(cmd, args)
-	os.RemoveAll(config.GetString("db_dir"))
-	os.Remove(config.GetString("cs_wal_file"))
+	ResetAll(config, log)
 }
 
 // XXX: this is totally unsafe.
 // it's only suitable for testnets.
 func resetPrivValidator(cmd *cobra.Command, args []string) {
+	ResetPrivValidator(config, log)
+}
+
+func ResetAll(c *viper.Viper, l log15.Logger) {
+	ResetPrivValidator(c, l)
+	dataDir := c.GetString("db_dir")
+	walFile := c.GetString("cs_wal_file")
+	os.RemoveAll(dataDir)
+	os.Remove(walFile)
+	l.Notice("Removed all data", "dir", dataDir)
+	l.Notice("Removed walfile", "file", walFile)
+}
+
+func ResetPrivValidator(c *viper.Viper, l log15.Logger) {
```

Comment:
hmm we need to export `ResetAll` for being able to import this code for the basecoin CLI (https://github.com/tendermint/basecoin/pull/64/files#diff-81718560b770949c7ec1743c3485f9df) but I don't believe we need to export ResetPrivValidator, I'll switch it. Thanks